### PR TITLE
New feat notifications

### PR DESCRIPTION
**Pull request by Jemeni11**

### CODE REVIEW

File: src/components/Notifications/index.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+				styles.NotificationListItem,
+				size === 'large' &&
+					'first:p-6 first:!scale-x-100 first:scale-y-100 first:rounded-xl first:shadow-[0px_4px_33px_0px_rgba(0,0,0,0.02)]'
+			)}
+		>
+			<div className="rounded-[38px] aspect-square p-4 border-2 border-[#F3F4F4] bg-[#F9FAFA] mr-6">
+				{type === 'job' ? <JobIcon /> : <CheckMonochromeIcon />}
+			</div>
+			<Flex.Column gap={6}>
+				<Text color="var(--text-black)" weight={600} fontSize="16px">
```
Comment (adenekan41) [inline src/components/Notifications/index.tsx:32]: You don't to add fsize 16, cus its 16 by default

File: src/components/Notifications/index.tsx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+	time,
+	type = 'job',
+	size = 'normal',
+}: NotificationListItemProps) {
+	return (
+		<div
+			className={classNames(
+				styles.NotificationListItem,
+				size === 'large' &&
+					'first:p-6 first:!scale-x-100 first:scale-y-100 first:rounded-xl first:shadow-[0px_4px_33px_0px_rgba(0,0,0,0.02)]'
```
Comment (adenekan41) [inline src/components/Notifications/index.tsx:25]: Please take all this away

File: src/components/Notifications/index.tsx [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+				</Text>
+				<Text color="#3F4C58" fontSize="12px" weight={600} className="mt-4">
+					{time} ago
+				</Text>
+			</Flex.Column>
+			<div className="justify-end">
+				<Flex.Row gap={8} className="justify-self-end justify-end">
+					{type === 'job' ? (
+						<>
+							<CallToAction outline>Save</CallToAction>
```
Comment (adenekan41) [inline src/components/Notifications/index.tsx:46]: for this lets make it size="sm"

File: src/components/Notifications/Notifications.module.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
+@use '@/styles/base/utils' as *;
+
+.NotificationsContainer {
+	font-family: var(--font-secondary);
```
Comment (adenekan41) [inline src/components/Notifications/Notifications.module.scss:4]: Stop doing this !!, font are always inherited there's no reason to do this

File: src/components/Notifications/Notifications.module.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+	padding-bottom: rem(32px);
+
+	.NotificationListContainer {
+		.NotificationListItem {
+			background: #fff;
+			width: 100%;
+			display: grid;
+			grid-template-columns: 72px 6fr 2fr;
+			transform: scaleX(0.95);
```
Comment (adenekan41) [inline src/components/Notifications/Notifications.module.scss:13]: let's not please with scale

File: src/components/Notifications/Notifications.module.scss [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+.NotificationsContainer {
+	font-family: var(--font-secondary);
+	padding-bottom: rem(32px);
+
+	.NotificationListContainer {
+		.NotificationListItem {
+			background: #fff;
+			width: 100%;
+			display: grid;
```
Comment (adenekan41) [inline src/components/Notifications/Notifications.module.scss:11]: a normal display flex would do the job here no ?

Comment (Jemeni11) [reply src/components/Notifications/Notifications.module.scss:11]: I started out with flex, but for some reason, justify-self: end wasn't working on the CallToAction container. 
Although now that I'm thinking about it, I could use only two divs then use space-between

**Review by adenekan41 — APPROVED** [review]

**Pull request merged**